### PR TITLE
Fix for ENTESB 11633 - setting limit to prevent timeout.

### PR DESCRIPTION
**Pull request by KurtStam**
https://issues.jboss.org/browse/ENTESB-11633

### CODE REVIEW

File: app/connector/sql/src/main/java/io/syndesis/connector/sql/common/DbMetaDataHelper.java [hunk 1]
```diff
@@ -163,6 +163,13 @@ public ResultSet fetchProcedures(final String catalog,
         }
     }
 
+    @SuppressFBWarnings("OBL_UNSATISFIED_OBLIGATION")
```
Comment (zregvart) [inline app/connector/sql/src/main/java/io/syndesis/connector/sql/common/DbMetaDataHelper.java]:
```suggestion
    @SuppressFBWarnings("OBL_UNSATISFIED_OBLIGATION") // needed for https://github.com/spotbugs/spotbugs/issues/432
```

**Comment by zregvart** [comment]
Would be cool to point to a spotbugs issue so when it's fixed and we review the `@SuppresWarnings` we can remove it. Other than that LGTM.

(We can merge without this, let me know and I can approve)

**Comment by KurtStam** [comment]
Does this need backporting to 1.8?

**Comment by heiko-braun** [comment]
@kurt not at this point anymore

**Pull request merged**